doc/api/cli.md

```diff
@@ -2911,6 +2920,7 @@ one is included in the list below.
 * `--experimental-async-context-frame`
 * `--experimental-default-type`
 * `--experimental-detect-module`
+* `--experimental-enable-transformation`
```

Comment:
Sugg: I think `enable-transformation` could be a bit open, maybe we could tie that down to types by calling this flag `--experimental-enable-type-transformation` or making it shorter with `--experimental-enable-type-transform`

Reply:
`--experimental-enable-type-transform` sounds good